packages/spor-design-tokens/scripts/build.ts

```diff
@@ -1,16 +1,20 @@
 import StyleDictionaryFactory from "style-dictionary";
+import config from "../config";
 import { elmFormatter } from "./formatters/elm/module";
-import { typescriptTypingsFormatter } from "./formatters/typescript/typings";
+import { cjsModuleFormatter } from "./formatters/typescript/cjs-module";
+import { esModuleFormatter } from "./formatters/typescript/es-module";
 import { reactNativeTypescriptTypingsFormatter } from "./formatters/typescript/rn-typings";
+import { typescriptTypingsFormatter } from "./formatters/typescript/typings";
+import { elmNameTransformer } from "./transforms/elm/name";
 import { pxTransformer } from "./transforms/size/px";
 import { pxToRemTransformer } from "./transforms/size/pxToRem";
-import { elmNameTransformer } from "./transforms/elm/name";
-import config from '../config';
 
 const styleDictionary = StyleDictionaryFactory.extend(config);
 
 // Register formatters
 // Read about formatters @ https://amzn.github.io/style-dictionary/#/formats
+styleDictionary.registerFormat(cjsModuleFormatter);
+styleDictionary.registerFormat(esModuleFormatter);
```

Comment:
Her registreres de to nye formattererne – slik at jeg kan referere til dem i `config.ts` lenger opp.